app-nuxt-base/server/api/stories.ts

```diff
@@ -0,0 +1,16 @@
+import type { Story } from '~/shared/types'
```

Comment:
Now the API endpoints don't need to deal with authentication, because the middleware takes care of it beforehand.